modules/images/fetchpriority/load.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Module Name: Fetchpriority
+ * Description: Adds `fetchpriority=high` parameter to the non lazy loaded images.
+ * Experimental: Yes
+ *
+ * @since n.e.x.t
+ * @package performance-lab
+ */
+
+/**
+ * Filters an image tag in content to add the fetchpriority attribute if it is not lazy-loaded.
+ *
+ * @since n.e.x.t
+ *
+ * @param string $filtered_image The image tag to filter.
+ * @param string $context        The context of the image.
+ * @return string The filtered image tag.
+ */
+function fetchpriority_img_tag_add_attr( $filtered_image, $context ) {
```

Comment:
Do we need to add a filter to disable this functionality? Like with Lazy Load, we have `add_filter( 'wp_lazy_loading_enabled', '__return_false' );` filter in the core.

Reply:
not here, but in core yes.
we can just turn it off in the plugin